deps/rabbit_common/src/supervisor2.erl

```diff
@@ -60,6 +60,7 @@
 %%
 -module(supervisor2).
 
+-compile(nowarn_deprecated_callback).
```

Comment:
We can safely drop this `format_status/2` as well instead. https://github.com/erlang/otp/blob/master/lib/stdlib/src/supervisor.erl doesn't implement it either.